### PR TITLE
Disable edit button and hide remove button if election is finished.

### LINKED ISSUE #3099: Possible to start new election round for finished elections
- go to an election that already took place. Its phase is therefore "Finished"
- click on "new ballot"

result: a new ballot is started. This must be impossible because after an election has been finished be the assembly leaders it cannot be changed. For political parties this is even a restriction in the German laws for political parties.

It is OK that one can reset the state from "Voting" to "Searching for candidates" because participants are allowed to enter a motion to change the voting process during the voting phase.
So to fix this bug, only the state "Finished" must not be resettable. If something went wrong one is forced to make a new election. Maybe other users of OS don't have such a strict rule. If so, there should be an option to block resetting the "Finished" phase.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
(Fixed #3099)

### CODE REVIEW

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 1]
```diff
@@ -133,7 +133,7 @@ <h3 translate>Election result</h3>
         index="$index" heading="{{ 'Ballot' | translate }} {{ $index + 1 }}">
       <div ng-controller="AssignmentPollDetailCtrl">
         <!-- action buttons -->
-        <div class="pull-right" os-perms="assignments.can_manage">
+        <div class="pull-right" os-perms="assignments.can_manage" ng-show="assignment.phase != 2">
```
Comment (normanjaeckel) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: You can use $scope.phases I think: `ng-show="assignment.phase != _.find(phases, {'display_name': 'Finished'}).value`

So we do not have a hard coded value here.

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 2]
```diff
@@ -158,6 +158,7 @@ <h3 translate>Election result</h3>
           </a>
           <!-- Edit -->
           <button ng-click="editPollDialog(poll, $index+1)"
+              ng-class="{ 'disabled': assignment.phase == 2 }"
```
Comment (normanjaeckel) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: Here the same.

File: openslides/assignments/static/templates/assignments/assignment-detail.html (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: I tested it: lodash function in template does not work. And I don't like the very long form (via 'display_name'). We already use the values  (0,1,2) in all code parts of this detail template an in pdf and projector. So: won't fix.

Comment (FinnStutzenstein) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: But: use `===` and `!==`.

Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: Done

File: openslides/assignments/static/templates/assignments/assignment-detail.html (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: Done

**Comment by emanuelschuetze** [comment]
Done and rebased.

**Pull request merged**